src/resources/js/events-admin.js

```diff
@@ -25,14 +25,14 @@ var tribe_timepickers = tribe_timepickers || {};
 	obj.$ = {};
 
 	obj.container = function( k, container ) {
-		var $container  = $( container );
-		var $all_day    = $container.find( obj.selector.all_day );
-		var $timepicker = $container.find( obj.selector.timepicker );
-		var $timezone   = $container.find( obj.selector.timezone ).not( obj.selector.input );
-		var $input      = $container.find( obj.selector.timezone ).filter( obj.selector.input );
+		var $container = $( container ),
+			$all_day = $container.find( obj.selector.all_day ),
```

Comment:
This isn't 'wrong', but see our notes on [variable declarations in JS](http://moderntribe.github.io/products-engineering/code-standards/javascript/#variable-declaration) - the previous code where each variable is declared in its own statement is usually what we aim for.